Bugfix for contact

DogukanGun opened the pull request:
**What is done:**

- [ ] Feature added
- [x] Bug fixed
- [ ] Code refactored
- [ ] Documentation updated
- [ ] Tests added
- [ ] Other (please specify):

### PR Purpose

#### Fixed Issues

<ul>
<li> Contact form responsiveness </li>
<li> Black text color whenever home page is returned</li>
<li> Fixing static width and height values </li>
</ul>

kihlaj commented:
![image](https://github.com/NexArb/WebApp/assets/65562071/97d511da-648d-4de5-b009-c1be2f0ae801)
![image](https://github.com/NexArb/WebApp/assets/65562071/31c75cb8-b8ba-4f82-bf23-61fcb8386c0f)
Now, it looks like this with the last commit. I suppose Dogukan didn't quiet finish it, yet. 

![image](https://github.com/NexArb/WebApp/assets/65562071/05fcad7b-297f-4ebc-895a-4725c8b9705b)

I moved text-white to /app/layout.tsx since then in everypage it is applied, so this is not needed. When you use it here it only applies to /. But for layout all pages styles. 

![image](https://github.com/NexArb/WebApp/assets/65562071/e8e924e0-fea8-4fec-8552-960bf28f34d8)

Image makes it really nice, for responsiveness increases the page load speed, by downloading specific sizes of the images. And by this why only small background showed. That is why in the picture it looks like that.

DogukanGun commented:
> ![image](https://private-user-images.githubusercontent.com/65562071/285652662-97d511da-648d-4de5-b009-c1be2f0ae801.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEwMDY5MjAsIm5iZiI6MTcwMTAwNjYyMCwicGF0aCI6Ii82NTU2MjA3MS8yODU2NTI2NjItOTdkNTExZGEtNjQ4ZC00ZGU1LWIwMDktYzFiZTJmMGFlODAxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI2VDEzNTAyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQxZTkxMTIyNmYwOGY5ZTRmNmU2NDZjZThlNWUxYzExZmI5MTExNDBlYjdlZTVmNjRlODlmZmY2M2U4MWE3ZWYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.dzMXszlLL-GLYwMBiN9dBo9LPhfuPoTO3eMx-1Ivmek) ![image](https://private-user-images.githubusercontent.com/65562071/285652696-31c75cb8-b8ba-4f82-bf23-61fcb8386c0f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEwMDY5MjAsIm5iZiI6MTcwMTAwNjYyMCwicGF0aCI6Ii82NTU2MjA3MS8yODU2NTI2OTYtMzFjNzVjYjgtYjhiYS00ZjgyLWJmMjMtNjFmY2I4Mzg2YzBmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI2VDEzNTAyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE5NGI1NWRlN2ZkYzFkMjVmYjQyZGY4MzRlNzRiNDI4YTM5OTVjZTg1ZDcyYjc4ZTkxYTljM2JmMjlkNjIxYzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-o_inx7hTO3V4rwG_cBSuiK_OU2mZmElyQ-cJRZo73I) Now, it looks like this with the last commit. I suppose Dogukan didn't quiet finish it, yet.
> 
> ![image](https://private-user-images.githubusercontent.com/65562071/285652831-05fcad7b-297f-4ebc-895a-4725c8b9705b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEwMDY5MjAsIm5iZiI6MTcwMTAwNjYyMCwicGF0aCI6Ii82NTU2MjA3MS8yODU2NTI4MzEtMDVmY2FkN2ItMjk3Zi00ZWJjLTg5NWEtNDcyNWM4Yjk3MDViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI2VDEzNTAyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI3YzY3ZjBmNjEyZTM4YzM5ZTViNWJlMzQ0NDVjZDAzMWEyODEwNzE4OTY3MmM4MWNjZDM0YjA3NWMxYjdjN2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-LiL2PUlfIpPlZOfmZocI3TGKrFjKeSwQVwBlP_B4pI)
> 
> I moved text-white to /app/layout.tsx since then in everypage it is applied, so this is not needed. When you use it here it only applies to /. But for layout all pages styles.
> 
> ![image](https://private-user-images.githubusercontent.com/65562071/285652929-e8e924e0-fea8-4fec-8552-960bf28f34d8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDEwMDY5MjAsIm5iZiI6MTcwMTAwNjYyMCwicGF0aCI6Ii82NTU2MjA3MS8yODU2NTI5MjktZThlOTI0ZTAtZmVhOC00ZmVjLTg1NTItOTYwYmYyOGYzNGQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzExMjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMTI2VDEzNTAyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdmNGFlMjM3OGFlNzcxZmUyYmMyMjEyNWI0NWM2NThiNTg2MDU1NGQ5NTQxZTMxZGIyOGU4OGQxMDc3NWYwM2YmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8c_yytkJHfoZ_dNAvX6Cp4b-7pgOy5vaig9SkmCR320)
> 
> Image makes it really nice, for responsiveness increases the page load speed, by downloading specific sizes of the images. And by this why only small background showed. That is why in the picture it looks like that.

I changed the images because in smallest screen like iPhone XR, background image is not shown well. For the `text-white` in layout, there was a bug. Whenever an user presses the back button, all texts in the page were black. I observed that, layout `text-white` is not working only in that case. Therefore I put the text-white inner component
